app/javascript/src/case_emancipation.js

```diff
@@ -62,6 +62,43 @@ function saveCheckState (action, checkItemId) {
     })
 }
 
+export function manageTogglerText ($parent) {
```

Comment:
**suggestion:** since these methods usually should act under the same parent, it might be worthwhile to wrap this in a class or using closures.

Something like
```js
class Toggler {
  constructor($parent) {
    this.$parent = $parent
  }

  manageTogglerText() {
    // your method using this.$parent
  }
}

const $category = $('selector')
const toggler = new Toggler($category)
toggler.manageTogglerText()
```

or

```js
function Toggler($parent) {
  const manageTogglerText = () => {
    // your method using $parent
  }

  return { manageTogglerText }
}

const $category = $('selector')
const toggler = Toggler($category)
toggler.manageTogglerText()
```

Reply:
I had forgotten about this, but updated it to use a class